app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -166,231 +169,122 @@ For the following Ukrainian and Afghan visas, you can apply for your visa from w
+
+* employed on a contract lasting less than one year
 
-## How trainee teachers can get the international relocation payment
+* employed by a recruitment agency – an agency can find and place you in a school, but your contract must be with the school and your salary must be paid directly to you by the school, not by the recruitment agency
 
-### Fee-paying trainees
+* teaching in Scotland, Wales or Northern Ireland 
 
-If you have a place on a teacher training course where you pay a fee, you do not need to apply for the international relocation payment (IRP). Your training provider will tell you if your choice of course and bachelor's degree make you eligible for the IRP. Your visa status and date of arrival will be checked by the Department for Education. If you are eligible, you’ll get the IRP around the end of your first term.
+## How to apply for the international relocation payment (IRP)
 
-If you believe you meet the eligibility requirements, but have not been informed by your teacher training provider that you may be awarded the IRP, please contact your training provider directly first. If your training provider cannot help you, you can email us at teach.inengland@education.gov.uk.
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) are open between 2 April and 31 May 2024. 
 
-### Salaried trainees
+You'll need to have started your teaching job to apply for the IRP. [Learn more about finding a job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas). You must apply in either the first or second term of your employment as a teacher.
 
-Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for salaried trainees are now closed. They will reopen between 2 April and 31 May 2024. 
+Eligible applicants who both start a job and apply for the IRP before 1 September 2024 will receive a single payment of £10,000 around the end of their first term  working in England.  
 
-To remain eligible for the IRP, you must apply in either the first or second term of your teacher training course. 
+Eligible applicants who apply for the IRP after 1 September 2024 will receive two £5,000 payments over 2 years, as long as they remain eligible. Teachers will be able to apply for the first payment at the start of their first year teaching in England and the second payment at the start of their second year. We’ll publish further details about how these split payments will be made to teachers on this page in due course. 
 
-If you have not yet started your salaried teacher training course, you can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We will email you when applications reopen later in 2024.
+You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We'll email you when applications reopen later in 2024.
```

Comment:
@laurahermionetennant92 do you need to delete the last sentence if applications have now reopened?

Reply:
They will open again in September but I could add for clarity